debug.log

```diff
@@ -0,0 +1,3 @@
+[0306/135352.106:ERROR:process_info.cc(359)] UncheckedAllocate
+[0306/135402.005:ERROR:process_info.cc(359)] UncheckedAllocate
+[0306/140047.563:ERROR:process_info.cc(359)] UncheckedAllocate
```

Comment:
I think this should not be pushed to git 😄 please remove and add `*.log` to `.gitignore`